images/openshift-applier/Dockerfile

```diff
@@ -6,7 +6,9 @@ ENV ANSIBLE_RPM http://releases.ansible.com/ansible/rpm/release/epel-7-x86_64/an
 USER root
 
 # Install Ansible and the 'oc' client
-RUN yum install -y $ANSIBLE_RPM ;\
+RUN INSTALL_PKGS="git" && \
```

Comment:
For consistency, and to make it stand out a bit more, can we please declare `INSTALL_PKGS` as an environment variable above?

Reply:
sure. I've gotten used to the way [SCL](https://github.com/sclorg/s2i-nodejs-container/blob/master/8/Dockerfile#L54) and [Jenkins](https://github.com/openshift/jenkins/blob/master/2/Dockerfile#L38) do it